examples/00_quick_start/lightgbm_tinycriteo.ipynb

```diff
@@ -628,20 +628,12 @@
    "execution_count": 7,
    "metadata": {},
    "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "/home/u/anaconda/envs/recommenders/lib/python3.9/site-packages/lightgbm/engine.py:181: UserWarning: 'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. Pass 'early_stopping()' callback via 'callbacks' argument instead.\n",
```

Comment:
checked that the code works with LightGBM 2.2.1 https://github.com/microsoft/LightGBM/blob/v2.2.1/python-package/lightgbm/callback.py. We can leave the dependency untouched